app/controllers/social_networking/nudges_controller.rb

```diff
@@ -41,20 +39,14 @@ def notify
       when "email"
         send_notify_email(@nudge, message_body)
```

Comment:
This method is looking up the recipient Participant again. Should you pass the Participant or Participant id instead of the Nudge? Also, `@nudge` is an instance variable so you don't need to pass it.